src/supervisor/subtree.rs

```diff
@@ -0,0 +1,23 @@
+use futures::future::{abortable, AbortHandle, Aborted, BoxFuture, Future, FutureExt};
```

Comment:
I added this file by accident, currently is not being included in the `lib.rs` file